Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidget.cxx

```diff
@@ -808,7 +862,7 @@ void vtkSlicerMarkupsWidget::Leave(vtkMRMLInteractionEventData* eventData)
       {
       interactionContext = eventData->GetInteractionContextName();
       }
-    markupsDisplayNode->SetActiveComponent(vtkMRMLMarkupsDisplayNode::ComponentNone, -1, interactionContext);
+    //markupsDisplayNode->SetActiveComponent(vtkMRMLMarkupsDisplayNode::ComponentNone, id, interactionContext);
```

Comment:
This dead code? Or accidental commented out code?

Reply:
Thanks @jamesobutler, this is dead code. I have just removed.